util/timer_test.cc

```diff
@@ -237,6 +237,49 @@ TEST_F(TimerTest, MultipleScheduleRepeatedlyTest) {
   ASSERT_EQ(count2, 5);
 }
 
+TEST_F(TimerTest, AddAfterStartTest) {
+  const int kIterations = 5;
+  uint64_t time_counter = 0;
```

Comment:
This line `uint64_t time_counter = 0` can be moved closer to where it is first used.